### ISSUE TITLE
The difference between "rotate" under Ubuntu and windows

### ISSUE THREAD

**Issue by WTG2017** [CadQuery/cadquery#130]
Hello, thank you for seeing my question. Fortunately, i have the opportunity to use your software in my work. One problem we found today is that “rotate” commands seem to be on different platforms.On ubuntu（conad3+pythonOCC+CQ-Editor）, rotate ((1, 0, 0), (0, 0), 90 * (math. pi / 180)) is the result I expected，On Windows 7（conda3+pythonOCC）, rotate ((1, 0, 0), (0, 0), 0, 90) can achieve the desired results.

**Comment by jmwright** [comment]
Hi @WTG2017 

Can you post a more complete example of what you're trying to do?

**Comment by WTG2017** [comment]
The same code, in different environments, has different results。
windows use angle：
![windows use angle](https://user-images.githubusercontent.com/28946294/58220490-b08eb700-7d41-11e9-86f8-64ab68678a96.png)
ubuntu use angle
![ubuntu use angle](https://user-images.githubusercontent.com/28946294/58220582-06635f00-7d42-11e9-9daf-77f41c3c87e9.jpg)
Can be modified to radian under ubuntu, the results are consistent
ubuntu use radian
![ubuntu使用弧度](https://user-images.githubusercontent.com/28946294/58220642-488ca080-7d42-11e9-898c-bd5bde9ba749.jpg)

**Comment by adam-urbanczyk** [comment]
Wow that is super weird @WTG2017, I cannot reproduce it though. Could you pip install latest cadquery version from git: `pip install git+https://github.com/CadQuery/cadquery.git` ?

**Comment by WTG2017** [comment]
Before this I have downloaded cadquery source code  from https://github.com/CadQuery/cadquery，and used the command “python setup.py install”to install it, but there has the "rotate problem" issue. Then I uninstalled it, and used the command "pip install git+https://github.com/CadQuery/cadquery.git" to reinstall it, but there has the issue still. Tomorrow I will  install a new version CQ-Editor on ubuntu and trying to find out which step went wrong.
